README.md

```diff
@@ -14,70 +14,173 @@ Takeaway Challenge
 
-Task
------
+## Instructions for use
+-------
+
+1. This program can be run from the Terminal/Command Line using IRB
+2. It requires Ruby 3.0.2, which can be installed with the following command:
+
+```
+rvm install 'ruby-3.0.2'
+```
+
+3. It also uses the Twilio API for sending SMS messages
+
+```
+gem install twilio-ruby
+```
+
+## User Stories
+-------
 
-* Fork this repo
-* Run the command 'bundle' in the project directory to ensure you have all the gems
-* Write a Takeaway program with the following user stories:
+The following user stories were supplied:
 
 ```
 As a customer
-So that I can check if I want to order something
-I would like to see a list of dishes with prices
+So that I can order the meal I want
+I would like to be able to select some number of several available dishes
+```
 
+```
 As a customer
 So that I can order the meal I want
 I would like to be able to select some number of several available dishes
+```
 
+```
 As a customer
 So that I can verify that my order is correct
 I would like to check that the total I have been given matches the sum of the various dishes in my order
+```
 
+```
 As a customer
 So that I am reassured that my order will be delivered on time
 I would like to receive a text such as "Thank you! Your order was placed and will be delivered before 18:52" after I have ordered
 ```
 
-* Hints on functionality to implement:
-  * Ensure you have a list of dishes with prices
-  * The text should state that the order was placed successfully and that it will be delivered 1 hour from now, e.g. "Thank you! Your order was placed and will be delivered before 18:52".
-  * The text sending functionality should be implemented using Twilio API. You'll need to register for it. It’s free.
-  * Use the twilio-ruby gem to access the API
-  * Use the Gemfile to manage your gems
-  * Make sure that your Takeaway is thoroughly tested and that you use mocks and/or stubs, as necessary to not to send texts when your tests are run
-  * However, if your Takeaway is loaded into IRB and the order is placed, the text should actually be sent
-  * Note that you can only send texts in the same country as you have your account. I.e. if you have a UK account you can only send to UK numbers.
+## My Approach
+-------
 
-* Advanced! (have a go if you're feeling adventurous):
-  * Implement the ability to place orders via text message.
+To start with, I created a menu_spec and test drove the functionality for browsing a menu as per the first user story:
 
-* A free account on Twilio will only allow you to send texts to "verified" numbers. Use your mobile phone number, don't worry about the customer's mobile phone.
+```
+jonnyabrams@JONNYs-MacBook-Pro takeaway-challenge % irb -r './lib/menu'
+3.0.2 :001 > menu = Menu.new
+ => 
+#<Menu:0x0000000142857058
+... 
+3.0.2 :002 > menu.browse
+bolognese: 7.5
+carbonara: 6.5
+puttanesca: 5.5
+arrabiata: 4.5
+ => {"bolognese"=>7.5, "carbonara"=>6.5, "puttanesca"=>5.5, "arrabiata"=>4.5} 
+```
+
```

Comment:
Seeing the output like this is really helpful - both in terms of following your process and having the expected working output visible